support-workers/src/main/scala/com/gu/support/workers/lambdas/Handler.scala

```diff
@@ -10,43 +10,42 @@ import io.circe.{Decoder, Encoder}
 import scala.concurrent.duration._
 import scala.concurrent.{Await, ExecutionContext, Future}
 
-abstract class Handler[T, R](implicit decoder: Decoder[T], encoder: Encoder[R]) extends RequestStreamHandler {
+abstract class Handler[T, R](
```

Comment:
I have folded Handler and FutureHandler together as the Handler was never used on its own.